src/command_modules/azure-cli-cognitiveservices/azure/cli/command_modules/cognitiveservices/commands.py

```diff
@@ -26,5 +24,8 @@ def load_command_table(self, _):
         g.command('regenerate', 'regenerate_key')
         g.command('list', 'list_keys')
 
+    with self.command_group('cognitiveservices account usage', mgmt_type) as g:
+        g.command('show', 'get_usages')
+
     with self.command_group('cognitiveservices') as g:
-        g.custom_command('list', 'list_resources', client_factory=cf_accounts)
+        g.custom_command('list', 'list_resources')
```

Comment:
Why isn't this under `accounts` group?

Reply:
I'm not sure about the original design decision.

But in our RP, we call this resource "account".
User create accounts and use account keys to access shared resources.

Reply:
This is non-blocking, but this command does return the accounts. It feels natural to me that `az cognitiveservice account list` returns accounts instead of `az cognitiveservice list`

Reply:
@troydai I agree with your feel.
so I deprecated the `az cognitiveservice list` and added `az cognitiveservice account list`.